lib/morse_translator.rb

```diff
@@ -40,4 +43,10 @@ def initialize
                     " " => " "}
   end
 
+   def translate(hash) 
+    @diccionary.map{|k,v| "#{k}=#{v}"}.join('&')
```

Comment:
You're right that this is a map, but you should be calling the map on each character of the input, what you labeled `hash` in this case.